app/feature.go

```diff
@@ -0,0 +1,97 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ExportFeatureFlag = "enable-export"
+)
+
+func GetFeatureFlagConfigPath() string {
+	return ConfigDirFlag.Value + "/feature.json"
```

Comment:
can you refactor to copy the login path logic:

```
configDir := ctx.Path(ConfigDirFlagName)
return filepath.Join(configDir, tokenFileName)
```

Reply:
Also does using the .Value notation work? I thought we usually use `ctx.Path` approach instead.

Reply:
Yeah. It works.